src/helpers/docker-compose-file.ts

```diff
@@ -0,0 +1,22 @@
+import { readFileSync, readdirSync, lstatSync } from "fs";
+import { join } from "path";
+import yaml from "yaml";
+import CONSTANTS from "../model/Constants.js";
+import { DockerComposeSpec } from "../model/DockerComposeSpec.js";
+
+export const getInitialDockerComposeFile = (path: string): DockerComposeSpec => {
+    return yaml.parse(
+        readFileSync(join(path, CONSTANTS.BASE_DOCKER_COMPOSE_FILE), "utf8").toString()
+    );
+};
+
+export const isFile: (fileName:string) => boolean = fileName => {
+    return lstatSync(fileName).isFile();
+};
```

Comment:
strictly speaking not really related to `docker-compose-file` I wouldn't think to
do

```ts
import { isFile } from `./helpers/docker-compose-file
```

Unless it was indicating whether it was a docker-compose spec file (which isn't what this function is doing)